src/js/azdata/azdataQueryPlan.js

```diff
@@ -362,7 +371,8 @@ azdataQueryPlan.prototype.init = function (container, iconPaths) {
                 }
             }
         }
-        //layout.execute(parent);
+        // Adding a very small cell to the parent for padding on the bottom right corner of the graph. 
+        vertex = graph.insertVertex(parent, 'paddingVertex', undefined, maxX+70+100, maxY+70+100, 0.0001, 0.0001, '');
```

Comment:
Why are calculations like `maxX+70+100` broken down like that? Seems simpler to just make it `maxX+170`. I'd also assign 170 to a const with an appropriate name, so it's easier to understand what it's meaning is.

Reply:
70 is the cell width and 100 is the padding. I will add 2 const variables for it.